hack/dind-cluster.sh

```diff
@@ -1021,7 +1057,17 @@ case "${1:-""}" in
       build-images "${OS_ROOT}"
     fi
 
-    refresh "${OS_ROOT}" "${CONFIG_ROOT}" "${CLUSTER_ID}" "${WAIT_FOR_CLUSTER}"
+    declare -a images=("")
+    if [[ -n "${ENABLE_LOCAL_REGISTRY}" ]]; then
+      error=0
+      images=$(get-local-images "${LOCAL_REGISTRY_IMAGES}") || error=1
```

Comment:
This line assigns a string value to `images`.  Why declare `images` as an array above and treat it as an array below?